src/megaclient.cpp

```diff
@@ -1159,6 +1159,13 @@ void MegaClient::exec()
                     case REQ_INFLIGHT:
                         if (pendingcs->contentlength > 0)
                         {
+                            if (fetchingnodes && fnstats.timeToFirstByte == NEVER
+                                    && pendingcs->bufpos > 10)
```

Comment:
It's intended to not set the value when error codes are received (usually `-3`).
A limit of 3 should be enough, but I have set it to 10 just in case there are bigger error codes in the future.